lib/zabbix/zabbix.go

```diff
@@ -203,3 +203,53 @@ func (a *API) GetTriggersInfo() (TriggerResults, error) {
 	}
 	return r.TriggerResults, nil
 }
+
+// AcknowledgeEvent acknowledges a single event and leaves a message.
+// It returns the event ID and an error if any.
+func (a *API) AcknowledgeEvent(eventID, message string) (int, error) {
+	params := struct {
+		EventIDs string `json:"eventids"`
+		// Event update action(s). This is bitmask field, any combination of values are acceptable.
+		// Possible values:
+		// 1 - close problem;
+		// 2 - acknowledge event;
+		// 4 - add message;
+		// 8 - change severity.
+		Action  int    `json:"action"`
+		Message string `json:"message"`
+	}{
+		EventIDs: eventID,
+		Action:   6,
```

Comment:
Should we be concerned that **6** is not in the list of accepted values :)

Reply:
6 is 4 (add message) + 2 (ack event)